Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -949,17 +949,17 @@ def aws_table_to_markdown(response, table_header):
     return human_readable
 
 
-def stringUnEscape(st):
+def stringEscape(st):
```

Comment:
We should leave the stringUnEscape as we don't want to break backwards. It could be that someone is using it.